src/app/query/bitcoin/address/utxos-by-address.hooks.ts

```diff
@@ -113,6 +137,33 @@ function useFilterInscriptionsByAddress(address: string) {
   };
 }
 
+function useFilterRuneUtxosByAddress(address: string) {
+  const { data, isLoading } = useRunesOutputsByAddress(address);
+  const runesEnabled = useRunesEnabled();
+
+  const filterOutRunesUtxos = useCallback(
+    (utxos: UtxoResponseItem[]) => {
+      // If Runes are not enabled, return all utxos
+      if (!runesEnabled) {
+        return utxos;
+      }
+
+      // To-DO If there are no runes response, filter out all utxos ?
+      if (!data) {
+        return [];
+      }
```

Comment:
Wonder if this is indeed too strict? won't this flash a balance of 0 before the data loads?